src/main/java/ChatReques5.java

```diff
@@ -0,0 +1,27 @@
+import lombok.*;
+
+import java.util.List;
+
+/**
+ * @author madhankumar
+ */
+@Setter
+@Getter
+@ToString
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class ChatReques5 {
+    private String model;
+    private List<?> messages;
+    private int n;
```

Comment:
Wildcard types like List<?> can cause issues with type safety. It is recommended to use specific types instead.